### DISCUSSION TITLE
problems running NG price sensitivity

### DISCUSSION THREAD

**Discussion by ssattler** [NREL/ReEDS-2.0#62] [Q&A: Execution] [answered]
I'm trying to run a high gas price scenario. I've used the AEO_2020_LOG setting for the ngscen switch in the cases file, but I'm getting an error in 1_Inputs when I start the run
269843  *written by input_processing\fuelcostprep.py
269844  table fprice(allt,r,f) "--2004$/MMBtu-- fuel prices by fuel type"
269847  $include inputs_case\fprice.csv
****                                  $282
**** LINE   2918 INCLUDE     C:\ReEDS_OpenAccess-master\runs\hiNG_ref_seq\b_inputs.gms
**** LINE      2 INPUT       C:\ReEDS_OpenAccess-master\runs\hiNG_ref_seq\createmodel.gms
269850  ;
****    $463
**** LINE   2921 INCLUDE     C:\ReEDS_OpenAccess-master\runs\hiNG_ref_seq\b_inputs.gms
**** LINE      2 INPUT       C:\ReEDS_OpenAccess-master\runs\hiNG_ref_seq\createmodel.gms
Can someone help? Thanks.

**Comment by vcelloho** [comment, marked as answer]
It looks like the fprice.csv file isn't being generated, this points to an issue with fulecostprep.py executing. Were you able to successfully run the model with default settings? One quick thing to check is whether there's an extra space or a typo in the ngscen setting. Another thing to check is whether there were any error messages generated when fuelcostprep.py ran in the log file.
